src/main/java/net/earthcomputer/clientcommands/command/KitCommand.java

```diff
@@ -0,0 +1,182 @@
+package net.earthcomputer.clientcommands.command;
+
+import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.arguments.StringArgumentType;
+import com.mojang.brigadier.exceptions.CommandSyntaxException;
+import com.mojang.brigadier.exceptions.DynamicCommandExceptionType;
+import com.mojang.brigadier.exceptions.SimpleCommandExceptionType;
+import com.mojang.brigadier.tree.LiteralCommandNode;
+import net.earthcomputer.clientcommands.interfaces.ISlot;
+import net.fabricmc.loader.api.FabricLoader;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.command.CommandSource;
+import net.minecraft.entity.player.PlayerInventory;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.nbt.ListTag;
+import net.minecraft.nbt.NbtIo;
+import net.minecraft.screen.slot.Slot;
+import net.minecraft.server.command.ServerCommandSource;
+import net.minecraft.text.TranslatableText;
+import net.minecraft.util.Util;
+
+import java.io.File;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static net.earthcomputer.clientcommands.command.ClientCommandManager.addClientSideCommand;
+import static net.earthcomputer.clientcommands.command.ClientCommandManager.sendFeedback;
+import static net.minecraft.server.command.CommandManager.argument;
+import static net.minecraft.server.command.CommandManager.literal;
+
+public class KitCommand {
+
+    private static final SimpleCommandExceptionType SAVE_FILE_EXCEPTION = new SimpleCommandExceptionType(new TranslatableText("commands.ckit.save.failed"));
+    private static final SimpleCommandExceptionType LOAD_FILE_EXCEPTION = new SimpleCommandExceptionType(new TranslatableText("commands.ckit.load.failed"));
+
+    private static final SimpleCommandExceptionType NOT_CREATIVE_EXCEPTION = new SimpleCommandExceptionType(new TranslatableText("commands.ckit.kit.notCreative"));
+    private static final DynamicCommandExceptionType NOT_FOUND_EXCEPTION = new DynamicCommandExceptionType(arg -> new TranslatableText("commands.ckit.kit.notFound", arg));
+
+    private static final Path path = FabricLoader.getInstance().getConfigDir().resolve("clientcommands");
+
+    private static final MinecraftClient client = MinecraftClient.getInstance();
+
+    private static final Map<String, PlayerInventory> kits = new HashMap<>();
```

Comment:
I think `PlayerInventory` stores a reference to the player, which stores a reference to the world. So when switching worlds you are going to get a memory leak because you're retaining a reference to the old world. Instead, you should store the `CompoundTag`s in this map.